datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -186,7 +186,7 @@ export class DatasetEntity implements Entity<Dataset> {
                     component: AccessManagement,
                     display: {
                         visible: (_, _1) => this.appconfig().config.featureFlags.showAccessManagement,
-                        enabled: (_, _2) => true,
+                        enabled: (_, dataset: GetDatasetQuery) => !!dataset?.dataset?.access,
```

Comment:
Good question. I assumed that the aspect wouldn’t be set if there are no roles. But I suppose it could happen that some job creates the aspect with an empty list of roles. I can add an additional condition.

Reply:
Added an additional check if the access aspect exists, but the roles property might be missing or an empty list.